packages/api/src/models/PaymentOptions.ts

```diff
@@ -0,0 +1,80 @@
+import { ErrorResponse, isProjectErrorResponse, transformErrorResponse } from './Error'
+
+/* eslint-disable @typescript-eslint/camelcase */
+
+export enum PaymentGatewayName {
+	WePay = 'fbpay',
+	Stripe = 'stripe',
+	PayPal = 'paypal',
+}
+
+export interface PaymentOptions {
```

Comment:
Can we add to https://github.com/freshbooks/freshbooks-nodejs-sdk/blob/main/packages/api/src/models/index.ts?

Reply:
Done 👍